docs/pages/access-controls/sso.mdx

```diff
@@ -350,20 +350,51 @@ fetching from a URL.
 </TabItem>
 <TabItem scope="oss" label="Teleport Community Edition">
```

Comment:
Do we need the outer "Edition" `Tabs` component? Nested tabs are difficult to navigate in our docs site. I would show only the "Dynamic Resources" method for both editions (with no edition tabs), explaining using prose that only the `github` type is supported in Teleport Community Edition. Since this isn't a comprehensive reference to configuring Teleport, I don't think we need to include the `auth_service` instructions.

Reply:
@ptgott please see update doing that.